mock_gps_node:
  ros__parameters:
    px4_micrortps: True  # Set to False for ArduPilot MAVROS
